src/zabapgit_unit_test.prog.abap

```diff
@@ -81,6 +81,28 @@ CLASS ltcl_convert IMPLEMENTATION.
 
 ENDCLASS.                    "ltcl_convert IMPLEMENTATION
 
+CLASS lth_critical_tests DEFINITION .
+  PUBLIC SECTION.
+    CLASS-METHODS:
+      check_run_permission.
+ENDCLASS.
+
+CLASS lth_critical_tests IMPLEMENTATION.
+
+  METHOD check_run_permission.
+    DATA: lo_settings TYPE REF TO lcl_settings.
+    lo_settings = lcl_app=>settings( )->read( ).
+
+    "Objects will be created and deleted, do not run in customer system!
+    "These tests may fail if you are locking the entries (e.g. the ZABAPGIT transaction is open)
+
+    IF lo_settings->get_run_critical_tests( ) = abap_false.
+      cl_abap_unit_assert=>fail( 'Cancelled' ).
```

Comment:
can we adjust this error message, so it tells the developer where to enable the tests? Or add it as a comment above